usaspending_api/recipient/v2/views/list_recipients.py

```diff
@@ -86,10 +133,22 @@ def post(self, request):
         ]
         models.extend(copy.deepcopy(PAGINATION))  # page, limit, sort, order
 
+        # Override pagination default limit of 100
+        for model in models:
+            if model["name"] == "limit":
+                model["max"] = 1000
+
         new_sort = {"type": "enum", "enum_values": ["name", "duns", "amount"], "default": "amount"}
         models = update_model_in_list(models, "sort", new_sort)
         models = update_model_in_list(models, "limit", {"default": 50})
         validated_payload = TinyShield(models).block(request.data)
 
-        results, page_metadata = get_recipients(filters=validated_payload)
+        try:
+            count = self.request_count(validated_payload)
+        except Exception as e:
+            count = None
+            logger.error("Unable to retreive count using endpoint. Falling back on using queryset.")
```

Comment:
Feel free to use something else. I was just thinking:
1. state _which_ endpoint
2. Since both methods are using a queryset, it is misleading to say a queryset will be used if there are problems

```suggestion
            logger.error("Unable to retrieve count using `/api/v2/recipient/duns/count`. Falling back on default behavior")
```